### DISCUSSION TITLE
Questions about translation feature from merge #136

### DISCUSSION THREAD

**Discussion by MenlyCSE** [EonEnglish/Eon-English-App#138] [Q&A] [answered]
## Some questions for @yusufshaikh719 
- Does this library this method require manual translation?
- If so, is this method preferable since we can make the translations ourselves or would you recommend exploring other options to translate the pages automatically and decrease maintenance?

**Comment by yusufshaikh719** [comment, marked as answer]
1. Yes, you need to add all your translations for the 3 languages
2. This method is much more cost effective and a lot faster, once the initial translations are done, there's nothing else, unlike other methods like dynamic translation where there is a character limit per API call. Another thing is that automatically translating might not be the translation you want, while manually doing it ensures that it is up to your standards.

**Reply by MenlyCSE** [reply]
This sounds amazing; fantastic work!
